rmw_connext_shared_cpp/src/node.cpp

```diff
@@ -70,6 +70,9 @@ create_node(
     participant_qos.user_data.value[sizeof(prefix) - 1 + name_length] = ';';
   }
 
+  // Setting the contentfilter_property_max_length to 1024
+  participant_qos.resource_limits.contentfilter_property_max_length = 1024;
```

Comment:
As it been confirm that we can still talk to native connext participants if we modify this default value ?

Reply:
Still investigating that to confirm along with the exact length for the property.

Reply:
I've checked native connext replier and requester, changed the `contentfilter_property_max_length` of requester to be 1024, replier to be 256 and vice-versa. The replier and requester worked as expected.